htdocs/PI/write/PIWriteRequest.php

```diff
@@ -902,14 +904,132 @@ private function updateEndpoint (Site $siteService) {
 
     #Make the requested change
     switch($this->entityProperty) {
+        case 'name':
+        case 'url':
+        case 'interfacename':
+        case 'description':
+        case 'email':
+        case 'monitored': {
+          switch ($this->requestMethod) {
+            case 'POST': {
+              $this->updateEndpointPropPost($endpoint, $this->entityProperty, $this->entityPropertyValue);
+              break;
+            }
+            case 'PUT': {
+              $this->updateEndpointPropPut($endpoint, $this->entityProperty, $this->entityPropertyValue);
+              break;
+            }
+            default: {
+              $this->exceptionWithResponseCode(405,$this->genericExceptionMessages["entityTypePropertyMethod"]);
+              break;
+            }
+          }
+          break;
+        }
         case 'extensionproperties':{
           $this->updateEndpointExtensionProperties($endpoint);
           break;
         }
         default: {
           $this->exceptionWithResponseCode(501,$this->genericExceptionMessages["entityTypePropertyCombo"]);
-        }
+      }
+    }
+  }
+
+  /**
+   * Update a property of an endpoint folloeing a POST requests
+   *
+   * @param  EndpointLocation $endpoint
+   * @param    $endpointPropName Name of property being updated
+   * @param    $endpointPropValue Value of property being updated
+   * @throws \Exception
+   */
+  private function updateEndpointPropPost (\EndpointLocation $endpoint, $endpointPropName, $endpointPropName) {
+    if($endpointPropName == 'monitored') {
+      #Not valid for boolians
```

Comment:
Why isn't `POST` valid for booleans?

Reply:
We don't have any null booleans - they are already defined as true or false. Post shouldn't overwrite existing values.

Reply:
```suggestion
      #Not valid for Endpoint booleans as they are set when entity created.
```

Reply:
review: for all comments update